src/components/pages/LoginPage/Login.module.scss

```diff
@@ -1,29 +1,76 @@
 .login-card {
   width: 100%;
-  margin: 10px auto;
+  margin: auto;
   text-align: center;
-  background: linear-gradient(transparent, #d2e7fc, transparent);
+  background: #001529;
```

Comment:
It is a good practice to define colors in variables and after use it in the code, for example 
@color-name1:  #001529;
@color-name2: #ED1234;
...
you can do it in scope of a separate task and update all scss